tools/wpt/utils.py

```diff
@@ -24,7 +24,11 @@ def call(*args):
     """
     logger.debug(" ".join(args))
     try:
-        return subprocess.check_output(args).decode('utf8')
+        # Since the ELF interpreter path of the webdriver binary may be
```

Comment:
Thanks for the change.

In the bug[1] you said the issue exists even if we launch chrome directly from command line, so is this the right place to fix it?

Also can we make this change directly in chromium by ignoring the presubmit error and trigger `linux-webkit-msan-rel` to see if this fixed the issue? Thanks!

[1] https://g-issues.chromium.org/issues/360008882#comment6